example/src/example-initial-value.json

```diff
@@ -311,6 +314,7 @@
               "50000": "#ef6548"
             }
           },
+          "dataApiUrl": "https://form-service.akvotest.org/api/answer-stats/1693403547388",
```

Comment:
https://akvo.github.io/akvo-react-form-editor/api/answer-stats
create a json file (but without json extension) under public/api/answer-stats

Reply:
sorry, you can just changed it to `https://akvo.github.io/akvo-react-form/api/meals`

Reply:
Hi mas @dedenbangkit , 
I have pushed a change. Please take a look when you have time.